app.rb

```diff
@@ -0,0 +1,149 @@
+require 'date'
+require_relative 'preserve'
+require_relative './game/game'
+require_relative './game/author'
+require_relative './books/book'
+require_relative './books/label'
+require_relative './music/genre'
+require_relative './music/music_album'
+
+class App
+  def initialize
+    @preserve = Preserve.new
+    @preserve.load_labels
+    @preserve.load_authors
+    @preserve.load_genres
+    @preserve.load_books
+    @preserve.load_games
+    @preserve.load_music_albums
+  end
+
+  def books_list
+    return puts 'No books found' if @preserve.books.empty?
+
+    @preserve.books.each_with_index do |book, index|
+      puts "#{index + 1}) Label: #{book.label.title}, Publisher: #{book.publisher}, Publish date: #{book.publish_date}"
+    end
+  end
+
+  def music_albums_list
+    return puts 'No music albums found' if @preserve.music_albums.empty?
+
+    @preserve.music_albums.each_with_index do |music_album, index|
+      puts "#{index + 1}) Genre: #{music_album.genre.name}, On spotify: #{music_album.on_spotify}"
+      puts "Publish date: #{music_album.publish_date}"
+    end
+  end
+
+  def games_list
+    return puts 'No games found' if @preserve.games.empty?
+
+    @preserve.games.each_with_index do |game, index|
+      puts "#{index + 1}) Author: #{game.author.first_name} #{game.author.last_name}"
+      puts "Multiplayer: #{game.multiplayer}, Last played: #{game.last_played_at}, Publish date: #{game.publish_date}"
+    end
```

Comment:
- [ ] [OPTIONAL] Because this is not specified in the UML diagram I will not mark it as a requirement however when adding a book, music album or a game it might be nice if they have titles because now only the title of the label, author or genre are displayed but the user will never know which book, music album or game it will be because there is no title.